public/JavaScript/gallery.js

```diff
@@ -0,0 +1,45 @@
+document.addEventListener('DOMContentLoaded', () => {
+  // Object mapping animal types to their respective classes
+  const animals = {
+    dogs: '.dog',
```

Comment:
Having logic tied to a CSS class should be avoided when possible. What other options can you think of?

Reply:
i can work with data attributes instead so im not tied to css anymore